<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>item_answer</class>
 <widget class="QWidget" name="item_answer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_itemans_text1,#label_itemans_ansUser,#label_itemans_date,#label_itemans_text2,#label_itemans_praisNum
{
    color: #868282;
}

#textBrowser_itemans_text{
	border: none;
}
#label{
	color: black;
}</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_itemans_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_itemans_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>132</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_itemans_text1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>9</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回答者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_itemans_ansUser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>12</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_itemans_text2">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>9</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>赞数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_itemans_praisNum">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>12</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>133</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>双击此处查看</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
